cli/dev/DevServer.ts

```
@@ -137,6 +149,56 @@ export class DevServer {
       .json()
   }
 
+  async handleFileRemovedFromFilesystem(absoluteFilePath: string) {
+    const relativeFilePath = path.relative(this.projectDir, absoluteFilePath)
+    try {
+      await this.fsKy.post("api/files/delete", {
+        json: {
+          file_path: relativeFilePath,
+          initiator: "filesystem_change",
+        },
+      })
+      console.log(`${relativeFilePath} deleted. Removed from file server.`)
+    } catch (err: unknown) {
+      console.error(
+        `Failed to remove file from file server: ${relativeFilePath}`,
+        err,
+      )
+    }
```

you're silencing this error, remove the try catch (same as the other ones)